tests/system/webdriver/tests/services/ConfigFrontEnd0002Test.php

```diff
@@ -76,8 +76,11 @@ public function tearDown()
 	 */
 	public function testChangeMetaDescription()
 	{
-		$this->siteHomePage->changeMetaDescription('JoomlaTestMetaDescription');
+		$this->assertEquals($this->previousMetaDescription, $this->siteHomePage->getMetaDescription(), 'Meta Description not changed');
```

Comment:
actually not required I would say, why to increase the execution time, when both of them will always have the same value every time
```
$this->siteHomePage->getMetaDescription()
```